### PR TITLE
Fix add item return of string / Add and Subtract of Quantity

### PR DESCRIPTION
**Pull request by EmTampz**

### CODE REVIEW

**Comment by EmTampz** [comment]
@wenyngcar @MiguelCanedo @kramnev

File: frontend/src/components/Add_subtractQuantity.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    row: any;
+    }
+
+const QuantityCell = ({ value, row }: QuantityCellProps) => {
+const [quantity, setQuantity] = useState(value);
+
+const handleIncrement = () => {
+    setQuantity(quantity + 1);
+    // Update the quantity in the backedn
+};
```
Comment (wenyngcar) [inline frontend/src/components/Add_subtractQuantity.tsx:17]: For consistency, use function declaration, not arrow function declaration.

File: frontend/src/components/Add_subtractQuantity.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const handleDecrement = () => {
+    if (quantity > 0) {
+    setQuantity(quantity - 1);
+    // Update the quantity in the backedn
+    }
+};
```
Comment (wenyngcar) [inline frontend/src/components/Add_subtractQuantity.tsx:24]: Do the same from review above.

File: frontend/src/components/columns.tsx [hunk 3]
```diff
@@ -33,3 +46,44 @@ export const columns: ColumnDef<Payment>[] = [
...
+      //   brandId={row.original.id}
+      //   brandName={row.original.brand}
+      //   brandCategory={row.original.category}
+      // />
+      <Button>Edit</Button>
+    ),
+  },
+];
+
+import QuantityCell from './Add_subtractQuantity';
```
Comment (wenyngcar) [inline frontend/src/components/columns.tsx:81]: Move import to the top.

File: frontend/src/components/Add_subtractQuantity.tsx
Comment (wenyngcar) [inline frontend/src/components/Add_subtractQuantity.tsx:1]: rename the file to add-subtract-quantity.tsx

**Review by wenyngcar — CHANGES_REQUESTED** [review]
No huge changes, request changes are for good practices and readability.

**Pull request closed without merging**